### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by henrijoss**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/housekeeping/update-deps

#### Scope

Short description of the pull request scope

#### Resolved Issue

Resolves [IssueNumber]

### CODE REVIEW

File: docs/dependencies.md [hunk 1]
```diff
@@ -5,37 +5,11 @@ reflected in the configuration file under `.github/dependabot.yml` and must be k
...
-| Next.js | 14.1.3   | 13.4.19 (React 18.2.0) |
-
-## Playwright
-
-Skipped `v1.43.0` & `v1.44.0` for now because ::before elements in dark mode are not visible on VRT screenshots.
-
-In case it gets updated, the Porsche Design System / Playwright Docker image needs to be updated too (this might affect
-all running workflows even outside the changed branch, so this must be aligned with the other developers).
-
-1. Open `./docker/Dockerfile` and adjust the Playwright docker image with the updated npm `@playwright/test` version,
```
Comment (stephanschroeter) [inline docs/dependencies.md:21]: Why have you removed the instructions on how to update the docker image / playwright?

File: docs/dependencies.md [hunk 2]
```diff
@@ -5,37 +5,11 @@ reflected in the configuration file under `.github/dependabot.yml` and must be k
...
-7. Manually execute "Run workflow"
-8. After docker image has been built by CI/CD, execute `./docker.sh bash` in the root directory of the repository
-9. Execute within Docker container `node --version` and update the Volta section of `./package.json` in the root
-   directory of the repository accordingly
-10. Execute within Docker container `yarn --version` and update the Volta section of `./package.json` in the root
-    directory of the repository accordingly
-11. Commit and push the changes to a Git branch
+|         | Monorepo | Sample Integrations   |
+| ------- | -------- | --------------------- |
+| Angular | 18.1.0   | 17.1.1                |
```
Comment (stephanschroeter) [inline docs/dependencies.md:10]: Are those versions still correct?

File: packages/components/scripts/patchStencilCore.ts [hunk 3]
```diff
@@ -29,6 +29,8 @@ const patchStencilCore = (): void => {
   const pdsPatchStartRegEx = new RegExp(`(${PDS_PATCH_START})`, 'g');
   const getPatchMarkerCount = (script: string): number => (script.match(pdsPatchStartRegEx) || []).length;
 
+  console.log(fileContent);
```
Comment (stephanschroeter) [inline packages/components/scripts/patchStencilCore.ts:32]: left over?

File: yarn.lock
Comment (stephanschroeter) [inline yarn.lock:1]: A lot of dependencies were added, can this be right?

File: packages/components-angular/package.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
     "build:subPackages:lib": "yarn rollup -c ./projects/angular-wrapper/rollup.config.js --bundleConfigAsCjs",
     "build:subPackages:typings": "yarn build:subPackages:typings:jsdom-polyfill && yarn build:subPackages:typings:partials && yarn build:subPackages:typings:testing && yarn build:subPackages:typings:styles",
     "build:subPackages:typings:jsdom-polyfill": "tsc projects/angular-wrapper/src/jsdom-polyfill/index.ts --declaration --emitDeclarationOnly --declarationDir dist/angular-wrapper/jsdom-polyfill --types node",
-    "build:subPackages:typings:partials": "tsc projects/angular-wrapper/src/partials/index.ts --declaration --emitDeclarationOnly --declarationDir dist/angular-wrapper/partials --types node",
+    "build:subPackages:typings:partials": "tsc projects/angular-wrapper/src/partials/index.ts --declaration --emitDeclarationOnly --declarationDir dist/angular-wrapper/partials --types node --skipLibCheck true",
```
Comment (stephanschroeter) [inline packages/components-angular/package.json:17]: Why do we have to add skip lib check?

File: docs/dependencies.md (hunk 2 above)
Comment (henrijoss) [reply docs/dependencies.md:10]: Yes just double checked

File: yarn.lock
Comment (henrijoss) [reply yarn.lock:1]: I always added one by one and reverted the yarn.lock and checked again

File: packages/components-angular/package.json (hunk 4 above)
Comment (henrijoss) [reply packages/components-angular/package.json:17]: Typescript error in the partials: Cannot find `JSX.Element` in namespace

**Review by stephanschroeter — APPROVED** [review]

**Pull request merged**